e2e/app/aou-elements/base-element-interface.ts

```diff
@@ -0,0 +1,30 @@
+import {ClickOptions, ElementHandle, WaitForSelectorOptions} from 'puppeteer';
+
+export interface BaseElementInterface {
```

Comment:
Maybe I missed some discussion of this - but is there a reason to separate this interface from the base class? Will this be used independently somehow?

Reply:
It was created for something else which has changed, so it is used only by `BaseElement` for now. I kept it in case it becomes useful in future. I don't have any strong feeling about keep or remove. either works.

Reply:
+1, I also wrinkled my nose at this but forgot to comment. In general, "in case it becomes useful for the future" is a personal warning flag for https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it. Asking myself that is often a really good gut-check to make sure I'm not over-thinking something, or building functionality that will later die on the vine.